src/components/faq/FaqSearch.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Box, InputAdornment, TextField } from '@mui/material'
+import SearchIcon from '@mui/icons-material/Search'
+import { useTranslation } from 'react-i18next'
+
+type Props = {
+  onChange: (newValue: string) => void
+}
+
+// Note: Consider reworking the main rendering logic so we hide the categories from the Tabs list
```

Comment:
Note: We may need to do a few more iterations to improve the UX. For example, we may consider reworking the main rendering logic so we hide the categories from the Tabs list if there are no any matching results within (only when using the search functionality) or at least putting some kind of alert to notify the user about this (like "There are no matching results within this category"). Maybe @atomov93 and the design team can advice on this :)

![image](https://user-images.githubusercontent.com/1233496/202910865-17178b4e-abef-4c7d-818f-fc28d4125cd7.png)

Reply:
We can add a simple alert to inform the users. The actual message can be as simple as "No results"

```tsx
<Alert variant="outlined" severity="info">
  {t('filters.noResults')}
</Alert>
```

Reply:
I tried with this approach, but still it was not perfect, because sometimes we may not have results in the currently selected category, but we force the user to go an click through all of them in order to find any results. Instead, I decided to slightly re-work the rendering for the categories tabs so they follow the filtering logic as well.

Now, when we try to filter the questions by typing into the search field, the categories on the left will update and we'll see only those who contains matching results. Once we don't have anymore matches, we'll see a nice fallback message. What do you think?

![image](https://user-images.githubusercontent.com/1233496/203106295-32244b99-1547-4c69-9cb4-c8269d540e54.png)
![image](https://user-images.githubusercontent.com/1233496/203106861-5e7cfef4-05b8-45cb-b42c-118a38e39f72.png)